symbolic-uplc/src/ZkFold/Symbolic/UPLC/Evaluation.hs

```diff
@@ -293,6 +362,12 @@ evalConstant (CG2 _)         = error "FIXME: UPLC BLS support"
 pair :: Sym c => SymValue t c -> SymValue u c -> SymValue (BTPair t u) c
 pair (SymValue p) (SymValue q) = SymValue (p, q)
 
+makeSingleton :: (Sym c) => SymValue u c -> SymValue (BTList u) c
+makeSingleton (SymValue xs) = SymValue $ L.singleton xs
+
+concatList :: (Sym c) => SymValue u c -> SymValue (BTList u) c -> SymValue (BTList u) c
```

Comment:
Maybe `consList` would be a bit better name, `concat` usually means concatenation of two lists